tests/functional/fixtures/authentication.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+__all__ = (
+    "user",
+    "login_user",
+    "with_logged_in_user",
+    "with_logged_in_staff_member",
+    "with_logged_in_admin",
+)
+
+
+@pytest.fixture
+def user(factories):
+    return factories.User()
+
+
+@pytest.fixture
+def login_user(db_session, app, user):
+    def login_user():
+        # This is the hash for `pass` used below
+        user.password = "$2b$12$21I1LjTlGJmLXzTDrQA8gusckjHEMepTmLY5WN3Kx8hSaqEEKj9V6"
+        db_session.commit()
```

Comment:
The commit here means the other fixtures which do things like `user.staff = True` will be persisted to the DB. It also keeps the hash and the value of the hash together. Previously it was a bit unclear why this was being set.